CHANGELOG.md

```diff
@@ -0,0 +1,1730 @@
+# Changelog
+
+<!-- <START NEW CHANGELOG ENTRY> -->
+
+## v1.15.2
+
+([full changelog](https://github.com/mwouts/jupytext/compare/28cc7de...1fd7290))
+
+### Merged PRs
+
+- Version 1.15.2 [#1128](https://github.com/mwouts/jupytext/pull/1128) ([@mwouts](https://github.com/mwouts))
+- Add support for RISE slides in JupyterLab [#1126](https://github.com/mwouts/jupytext/pull/1126) ([@fcollonval](https://github.com/fcollonval))
+- Version 1.15.1 [#1120](https://github.com/mwouts/jupytext/pull/1120) ([@mwouts](https://github.com/mwouts))
+
+### Contributors to this release
+
+([GitHub contributors page for this release](https://github.com/mwouts/jupytext/graphs/contributors?from=2023-08-26&to=2023-09-16&type=c))
+
+[@fcollonval](https://github.com/search?q=repo%3Amwouts%2Fjupytext+involves%3Afcollonval+updated%3A2023-08-26..2023-09-16&type=Issues) | [@mwouts](https://github.com/search?q=repo%3Amwouts%2Fjupytext+involves%3Amwouts+updated%3A2023-08-26..2023-09-16&type=Issues) | [@parmentelat](https://github.com/search?q=repo%3Amwouts%2Fjupytext+involves%3Aparmentelat+updated%3A2023-08-26..2023-09-16&type=Issues)
```

Comment:
They show up quite nicely -> https://github.com/jupyterlab/jupyterlab-github/blob/main/CHANGELOG.md
It would be shame to remove them as it is nice to have contributors list.

Reply:
I was more concerned about `RTD` documentation, with how it handles the headers, etc. The complicated links seem to have some value, so yeah indeed it would be nice. But let's see how it is created there, and maybe we can adjust it a little bit, e.g. It might be better with a `---` separator instead of `###` headings.

PS: Can you remove the bot account contributions from the list?